### ISSUE TITLE
How can I fix warning Blocked a Frame with Origin

### ISSUE THREAD

**Issue by yendrapriambada** [gree/unity-webview#941]
The warning show like in this image:
![image](https://github.com/gree/unity-webview/assets/73283816/11e2c9f6-73c3-4dbf-b923-60d507a392b3)

Why the warning show and how to fix it?
Thanks Before!

**Comment by KojiNakamaru** [comment]
As discussed in https://github.com/gree/unity-webview/issues/507#issuecomment-583920773 , the cross-origin access is restricted. If both the page running the unity webgl content and the page displayed in iframe with this plugin belong to the same origin, you can avoid this error.

**Comment by yendrapriambada** [comment]
is there a way to hide the warning popup?

**Comment by KojiNakamaru** [comment]
This is an error, not a warning. As noted above, if pages are in the same origin, there will be no error. Also, if you don't use EvaluateJS() and simply display the page, there will be no error.

**Comment by yendrapriambada** [comment]
Yes, If I just want to display the page, how do i disable EvaluateJS?

**Comment by KojiNakamaru** [comment]
The sample app calls EvaluateJS() as below just for demonstrating it.

https://github.com/gree/unity-webview/blob/6b49b82199304381a43fb2bd3ec70833c242cd28/sample/Assets/Scripts/SampleWebView.cs#L113

If you don't call it, it won't cause the issue here.

**Comment by yendrapriambada** [comment]
Thankyou for your help!

**Comment by ssmore101** [comment]
Thank you!! It Worked
